st2common/st2common/config.py

```diff
@@ -500,6 +500,28 @@ def register_opts(ignore_errors=False):
         dispatcher_pool_opts, group="actionrunner", ignore_errors=ignore_errors
     )
 
+    graceful_shutdown_opts = [
+        cfg.BoolOpt(
+            "graceful_shutdown",
+            default=False,
```

Comment:
I think we should enable this by default, shouldn't we? Maybe this should default to true when the coordination backend is configured.